src/main/java/pl/wedding/management/converter/Converter.java

```diff
@@ -0,0 +1,33 @@
+package pl.wedding.management.converter;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class Converter<T, U> {
+
+    private final Function<T, U> fromDto;
+    private final Function<U, T> fromEntity;
+
+    public Converter(final Function<T, U> fromDto, final Function<U, T> fromEntity) {
+        this.fromDto = fromDto;
+        this.fromEntity = fromEntity;
+    }
+
+    public final U convertFromDto(final T dto) {
+        return fromDto.apply(dto);
+    }
+
+    public final T convertFromEntity(final U entity) {
```

Comment:
Ta klasa nie robi nic innego jak wywołuje inną metodę z klasy, która po niej dziedziczy. Niech w takim razie będzie to metoda abstrakcyjna.